pkg/config/config.go

```diff
@@ -17,17 +17,18 @@ import (
 const DefaultAppName = "content-sources"
 
 type Configuration struct {
-	Database   Database
-	Logging    Logging
-	Loaded     bool
-	Certs      Certs
-	Options    Options
-	Kafka      event.KafkaConfig
-	Cloudwatch Cloudwatch
-	Metrics    Metrics
-	Clients    Clients `mapstructure:"clients"`
-	Mocks      Mocks   `mapstructure:"mocks"`
-	Sentry     Sentry  `mapstructure:"sentry"`
+	Database         Database
+	Logging          Logging
+	Loaded           bool
+	Certs            Certs
+	Options          Options
+	Kafka            event.KafkaConfig
+	Cloudwatch       Cloudwatch
+	Metrics          Metrics
+	Clients          Clients `mapstructure:"clients"`
+	Mocks            Mocks   `mapstructure:"mocks"`
+	Sentry           Sentry  `mapstructure:"sentry"`
+	NewTaskingSystem bool    `mapstructure:"new_tasking_system"`
```

Comment:
you have to set a 'default' below for this or else it won't work as an env variable... its super annoying :(